### PR TITLE
Roll out Shortcuts by Default to 100% of Nightly & Beta on Desktop

### PR DESCRIPTION
**Pull request by fallaciousreasoning**
Staging PR: https://github.com/brave/brave-variations/pull/644

### CODE REVIEW

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers. QA has verified the PR on `main` via `Nightly`, `BETA` & `Release` as per https://github.com/brave/brave-variations/pull/644#issuecomment-1588394588. QA will run through the feature more thoroughly via `1.53.x`.

**Comment by fallaciousreasoning** [comment]
Confirmed this is showing up in Beta! Thanks @kjozwiak !

**Pull request merged**
